imports/ui/components/SearchBar.jsx

```diff
@@ -0,0 +1,294 @@
+import React from 'react';
+import clsx from 'clsx';
+import Select from 'react-select';
+import { emphasize, makeStyles, useTheme } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import NoSsr from '@material-ui/core/NoSsr';
+import TextField from '@material-ui/core/TextField';
+import Paper from '@material-ui/core/Paper';
+import Chip from '@material-ui/core/Chip';
+import MenuItem from '@material-ui/core/MenuItem';
```

Comment:
We can move all these /core imports into one ```import {Typography, NoSsr, TextField, Paper, Chip, MenuItem} from '@material-ui/core';``` to make adding more core items easier